mmv1/templates/terraform/constants/cloud_run_service.go.erb

```diff
@@ -3,6 +3,24 @@ func revisionNameCustomizeDiff(_ context.Context, diff *schema.ResourceDiff, v i
 	if autogen && diff.HasChange("template.0.metadata.0.name") {
 		return fmt.Errorf("google_cloud_run_service: `template.metadata.name` cannot be set while `autogenerate_revision_name` is true. Please remove the field or set `autogenerate_revision_name` to false.")
 	}
+
+	if diff.HasChange("template.0.metadata.0.annotations") {
```

Comment:
I think this gets the job done, but we already have a working list of annotations we suppress in the `cloudRunGoogleProvidedAnnotations` regex. We should just be able to add `sandbox` as an option to the `run.googleapis.com` part. See the `serving.knative.dev` part which holds two suffixes already (`creator` and `lastModifier`)

Reply:
I think `cloudRunGoogleProvidedAnnotations` is used for annotations at the service level, this one is for template level annotations. I have to reconstruct the regex for this level separately. 
Also I was just going with approach guided here - https://github.com/hashicorp/terraform-provider-google/issues/8568. If we want to change it to regex approach, i can do it.  Let me know your thoughts

Reply:
I see, thanks for the explanation! Then I would request we move this logic to its own customize diff function as `revisionNameCustomizeDiff` serves a different purpose. Along with that, it would be nice to have a unit test for the new customize diff function.

Reply:
As discussed, `customizediff` has been replaced with `dsf ` for simplicity. Thank you @c2thorn.